src/types.h

```diff
@@ -302,7 +303,7 @@ class SizedType {
 
   const std::string GetName() const
   {
-    assert(IsRecordTy());
+    assert(IsRecordTy() || IsIntTy());
```

Comment:
Nit. Does this work?
```suggestion
    assert(IsRecordTy() || IsEnumTy());
```

Reply:
Ah yeah that's better